homeassistant/components/cast/media_player.py

```diff
@@ -88,24 +91,63 @@ def is_audio_group(self) -> bool:
     @property
     def is_information_complete(self) -> bool:
         """Return if all information is filled out."""
-        return all(attr.astuple(self))
+        return (
+            (not self.is_audio_group or
+             self.is_dynamic_group is not None) and
+            all(attr.astuple(
```

Comment:
What is happening here. Are we just testing that any field is filled in, but then in a very complicated unreadable way? 😛

Reply:
Yeah.. I think it will look nicer if it's broken up.